docs/our_work/pets.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: base
+title: Emerging Privacy Enhancing Technologies
+permalink: pets.html
+summary: Investigating how different emerging PETs may have opportunity for healthcare data
+tags: ['PETs', 'Differential Privacy', 'Federated', 'Encryption']
+---
+
+Privacy Enhancing Technologies (PETs) are an emerging set of technologies and approaches that enable the derivation of useful results from data without providing full access to the data.   Whilst traditional de-identification of data reduces the utility as the privacy increases, PETs aim to keep the privacy very high whilst gaining a high enough level of utility that complex functions can be applied to the data.  
+
+To achieve a high confidence in the privacy of data we can either turn to a hardware approach or alternatively explore different statistical algorithms.   The hardware approach includes the use of trusted execution environments or enclaves which have absolute control on all the inputs and outputs into a space and thus can be used for secure controlled computation.   The algorithmic approach aims to allow certain operations on the accessible data which can be proven to have absolute or very high privacy.   These include Homomorphic Encryption and Secure Multi-Party Computation.   In addition output privacy can also be increase through the use of differential privacy to add a statistically significant level of noise making raw data inference impossible.  See the [CDEIUK PETs](https://cdeiuk.github.io/pets-adoption-guide/what-are-pets) adoption guide for more details.
+
+![Not this sort of PET](images/pets.png)
+
+The Royal society report *“From privacy to partnership The role of privacy enhancing technologies in data governance and collaborative analysis”* [1](https://royalsociety.org/-/media/policy/projects/privacy-enhancing-technologies/From-Privacy-to-Partnership.pdf?la=en-GB&hash=4769FEB5C984089FAB52FE7E22F379D6) states that
+
+> “Recent advances in medical imaging, audio and AI have led to unprecedented possibilities in healthcare and research. This is especially true of the UK, where the public health system is replete with population-scale electronic patient records. These conditions, coupled with strong academic and research programmes, mean that the UK is well positioned to deliver timely and impactful health research and its translation to offer more effective treatments, track and prevent public health risks, utilising health data to improve and save lives [2](https://www.gov.uk/government/publications/lifesciences-industrial-strategy-update).
+> ….
```

Comment:
in order for this to render as two paragraphs it just needs a blank line rather than this > ...